docs/src/5.1.MigratingToVersion0.2.x.mdx

```diff
@@ -0,0 +1,91 @@
+# Migrating from 0.1.x to 0.2.x
+
+The 0.2.1 Worldview release brought a new API for managing interactions and a variety of changes to how interactions are done. This is a breaking change - code that worked under 0.1.x may silently fail under 0.2.x. Below is a summary of what changed and how to update your own code.
+
+## Mouse event changes
+
+Previously, the Worldview-level event handler only passed an `objectId`, not the full object being clicked. Now both the Worldview-level event handlers and Command-level event handlers pass the same information about the click: the ray, and an array of objects that contain the object being clicked and the `instanceIndex`, if the clicked object is part of an instanced command.
+
+We now pass an array of objects because stacked object events can now optionally be enabled: passing `enableStackedObjectEvents={true}` to the `Worldview` component allows seeing all objects at the point the user clicked on, instead of just the top-level object.
```

Comment:
Have you thought about using a different prop `maxStackedObjectCount` to replace `enableStackedObjectEvents` . e.g. default `maxStackedObjectCount` to 0, which is the equivalent of `enableStackedObjectEvents=false`

Reply:
Yeah, I didn't get around to merging the two concepts. Even though this one is more verbose, it might be more understandable? not sure.

Reply:
What about just calling it `maxObjectCount` to make it clear it's related to the `objects` that you get back in an event handler? Then it make sense to have it default to 1.